lib/client/client.d.ts

```diff
@@ -89,7 +95,24 @@ declare namespace Client {
         /**
          * enable and/or configure automatic result paging
          */
-        paging?: boolean;
+        paged?: boolean | Paging;
+    }
+
+    export interface SearchResult {
+        status: number;
+        controls: Control[];
+    }
+
+    export interface SearchResponseEmitter extends EventEmitter {
+        on(ev: string, callback: (entry: SearchEntry) => void): this;
+        on(ev: string, callback: (referral: any) => void): this;
+        on(ev: string, callback: (err: LDAPError) => void): this;
+        on(ev: string, callback: (result: SearchResult) => void): this;
```

Comment:
Hmm, I'm not entirely sure why I left those in. Looks like an oversight. I deleted them from my local types and my code still compiled.